packages/react-native/src/primitives/Radio/styles.ts

```diff
@@ -0,0 +1,44 @@
+import { StyleSheet } from 'react-native';
+
+import { RadioStyles } from './types';
+
+export const styles: RadioStyles = StyleSheet.create({
+  container: {
+    alignItems: 'center',
+  },
+
+  disabled: {
+    opacity: 0.6,
+  },
+
+  radioButton: {
+    backgroundColor: '#317d95',
+    borderRadius: 999,
+  },
+
+  radioButtonContainer: {
+    alignItems: 'center',
+    borderColor: '#838c95',
+    borderRadius: 999,
+    justifyContent: 'center',
+  },
+
+  // radioButtonContainer sizes
+  large: {
```

Comment:
What do you think of removing these "size" related objects and returning the style objects from the dimension creation util?

Reply:
We could do that, but I'm wondering how customers would then be able to override the styles for a particular size (e.g., `large`).

Reply:
They would just pass the value they want in the `size` prop?

Reply:
Un-resolving this comment. Lmk if we need to discuss offline